### PR TITLE
[Internal] Add check for tags field (array)

### PR DESCRIPTION
**Pull request by optout21**
Add support for tags field, an array of values, from a predefined list of valid values.

### CODE REVIEW

File: script/generic/asset-infos.ts [hunk 1]
```diff
@@ -67,7 +68,14 @@ function isAssetInfoValid(info: unknown, path: string, address: string, chain: s
...
     // status
     if (!isValidStatusValue(info['status'])) {
-        return [`Invalid value for status field, '${info['status']}'`, "", fixedInfo]
+        return [`Invalid value for status field, '${info['status']}'`, "", fixedInfo];
+    }
+
+    // tags
+    if (info['tags']) {
+        if (!isValidTagValues(info['tags'])) {
+            return [`Invalid tags, '${info['tags']}'`, "", fixedInfo];
```
Comment (vikmeup) [inline script/generic/asset-infos.ts:77]: Can you check that tag contain in tags.json?

Comment (catenocrypt) [reply script/generic/asset-infos.ts:77]: That's what function isValidTagValues does

**Review by vikmeup — APPROVED** [review]

**Comment by vikmeup** [comment]
@catenocrypt lgtm, there is CI error, let's merge this

**Comment by catenocrypt** [comment]
With 'var' Lint gave an error, with 'const' it doesn't compile LOL

**Pull request merged**